app/models/featured_resource.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class FeaturedResource < ApplicationRecord
```

Comment:
I think you should use an activerecord polymorphic association here, with a foreign key constraint in the database. You're essentially reinventing that wheel via the UUID, but the UUID columns don't have any sort of key constraints on them. That will also allow you to load all the featured resources out in one database call in PagesController as well.

Reply:
@rschenk I thought about that, except there's only going to be a few of these in here. The pages controller only needs to load three. That seemed like over-optimization for such a small set of things. But, I can go that way if you think it's the better solution.

There's the other issue of sorting. I provided a sort field, but I think I need to provide some kind of constraint that there can only be one record with a sort value of "1". It's also getting quite overly-complex for a solution that only needs to support three featured works at a time. But I don't know of any other way to do it. 🤷‍♂️